javaslang-benchmark/src/test/java/javaslang/benchmark/JmhRunner.java

```diff
@@ -1,94 +0,0 @@
-package javaslang.benchmark;
```

Comment:
I moved these to the default package to have access to the helper classes.
I committed it separately, without modifications. It's strange that Github doesn't recognize the moves :/